man/ocamlopt.1

```diff
@@ -540,12 +540,18 @@ types are derived in a principal way. All programs accepted in
 .B \-principal
 mode are also accepted in default mode with equivalent
 types, but different binary signatures.
+.B \-no-principal
+Do not check principality of type inference.
+This is the default.
 .TP
 .B \-rectypes
 Allow arbitrary recursive types during type-checking.  By default,
 only recursive types where the recursion goes through an object type
-are supported. Note that once you have created an interface using this
-flag, you must use it again for all dependencies.
+are supported.
```

Comment:
This note should neither be removed from `ocamlc.1` and `ocamlopt.1` nor added to `ocaml.1`.